### DISCUSSION TITLE
Open current game folder?

### DISCUSSION THREAD

**Discussion by Maltavius** [Heroic-Games-Launcher/HeroicGamesLauncher#3464] [Ideas]
So I'm trying to find out where I've told Heroic Launcher I have my BG3 installed. But I can't seem to find a way to get Heroic to show me the path it's installed on. 

I can move it or change the installation path, but that only opens the default games folder.

**Comment by imLinguin** [comment]
You can find the information on the game page under `Installed Information`

![obraz](https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/assets/62100117/1852c070-f569-47d0-bd79-8ced4b2cb1b6)
